### PR TITLE
landing page

### PR DESCRIPTION
**Pull request by DadhichTanishka**
- landing page using react 
- and common navbar for all the pages

### CODE REVIEW

**Review by PatelNeelMahesh — APPROVED** [review]
You should remove the default readme file and update your project description and figma link and should update it as you add more features.

In src folder for you can remove default testing files.

**Comment by PatelNeelMahesh** [comment]
You should give better commit message.
Instead of "first pull" you can write "Initialized react app, created landing page & navbar"

File: frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
```
Comment (aditya-190) [inline frontend/README.md]: Update your README  -

Add  brief description about your project.
You can also add a checklist that can be used to track all the topics covered until now.

For reference - https://github.com/kalviumcommunity/openbadgx/blob/main/README.md

File: frontend/package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "capstone",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
```
Comment (aditya-190) [inline frontend/package.json]: It is advisable to include testing libraries in the devDependencies section, as failing to do so may result in their being bundled into the React application, consequently leading to an increase in the overall bundle size.

Reference -> https://www.geeksforgeeks.org/difference-between-dependencies-devdependencies-and-peerdependencies/

File: frontend/package.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "capstone",
```
Comment (aditya-190) [inline frontend/package.json]: Update the name to ClosetGenie

File: frontend/public/manifest.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (aditya-190) [inline frontend/public/manifest.json]: Update the name to ClosetGenie

File: frontend/src/App.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import './App.css';
+import {BrowserRouter as Router,Route, Routes} from 'react-router-dom';
+import Home from './home';
+import About from './aboutus';
+import Contact from './contactus';
+import Main from './homeMain';
+// import NavBar from './navbar';
```
Comment (aditya-190) [inline frontend/src/App.js]: Please remove the redundant comments.

**Review by aditya-190 — CHANGES_REQUESTED** [review]
Please address the comments before merging. Everything else LGTM.

**Comment by DadhichTanishka** [comment]
I have made the changes please check them.

**Review by aditya-190 — APPROVED** [review]
LGTM. Proceed with merging

**Pull request merged**